### PR TITLE
Adds help text for snapshot

### PR DESCRIPTION
**Pull request by Naveenjnv22**
Signed-off-by: naveen kumar <kumarnjnv22@gmail.com>

**What this PR does / why we need it**:This will give helptext for snapshot in the mayactl

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes https://github.com/openebs/maya/issues/348

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by Naveenjnv22** [comment]
```
Usage: mayactl  snapshot <subcommand> [options] [args]

	This command provides operations related to a snapshot.

	Create a snapshot:
	$ mayactl snapshot create --volname <vol> --snapname <snap>

	List snapshot:
	$ mayactl snapshot list --volname <vol>

	Revert a snapshot:
	$ mayactl snapshot revert --volname <vol> --snapname <snap>

```

File: cmd/mayactl/app/command/snapshot/create.go [hunk 1]
```diff
@@ -25,6 +25,18 @@ import (
...
+var (
+	snapshotCreateCommandHelpText = `
+	Usage: mayactl snapshot create [options]
+
+	$ mayactl snapshot create --volname <vol> --snapname <snap>
+
+
+	This command creates a new snapshot.
+
+	`
```
Comment (prateekpandey14) [inline cmd/mayactl/app/command/snapshot/create.go]: Remove extra spaces

File: cmd/mayactl/app/command/snapshot/list.go [hunk 2]
```diff
@@ -25,14 +25,24 @@ import (
...
 )
 
+var (
+	snapshotListCommandHelpText = `
+	Usage: mayactl snapshot list [options]
+
+	$ mayactl snapshot list --volname <vol>
+
+	This command displays status of available snapshot.
+	`
```
Comment (prateekpandey14) [inline cmd/mayactl/app/command/snapshot/list.go]: remove extra spaces

File: cmd/mayactl/app/command/snapshot/list.go [hunk 3]
```diff
@@ -25,14 +25,24 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	snapshotListCommandHelpText = `
+Usage: mayactl snapshot list [options]
+
+$ mayactl snapshot list --volname <vol>
+
+This command displays status of available snapshot.
```
Comment (kmova) [inline cmd/mayactl/app/command/snapshot/list.go]: This command displays available snapshots on a volume.

File: cmd/mayactl/app/command/snapshot/revert.go [hunk 4]
```diff
@@ -24,6 +24,16 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	snapshotrevertHelpText = `
+Usage: mayactl snapshot revert [options]
+
+$ mayactl snapshot revert --volname <vol> --snapname <snap>
+
+This command displays status of available snapshotrevert.
```
Comment (kmova) [inline cmd/mayactl/app/command/snapshot/revert.go]: This command rolls back the volume data to the specified snapshot. Once the roll back to snapshot is successful, all the data changes made after the snapshot was taken will be post. This command should be used cautiously and only when there is an issue with the current state of the data.

**Comment by Naveenjnv22** [comment]
`
snapshot revert
`

**Comment by Naveenjnv22** [comment]
snapshot revert

File: cmd/mayactl/app/command/snapshot/revert.go [hunk 5]
```diff
@@ -24,6 +24,18 @@ import (
...
 )
 
+var (
+	snapshotrevertHelpText = `
+Usage: mayactl snapshot revert [options]
+
+$ mayactl snapshot revert --volname <vol> --snapname <snap>
+
+This command rolls back the volume data to the specified snapshot. 
+Once the roll back to snapshot is successful, all the data changes made after the snapshot was taken will be post.
```
Comment (mahebbar) [inline cmd/mayactl/app/command/snapshot/revert.go]: ..., ...will be posted.

File: cmd/mayactl/app/command/snapshot/revert.go [hunk 6]
```diff
@@ -24,6 +24,18 @@ import (
...
 
+var (
+	snapshotrevertHelpText = `
+Usage: mayactl snapshot revert [options]
+
+$ mayactl snapshot revert --volname <vol> --snapname <snap>
+
+This command rolls back the volume data to the specified snapshot. 
+Once the roll back to snapshot is successful, all the data changes made after the snapshot was taken will be post. 
+This command should be used cautiously and only when there is an issue with the current state of the data.
```
Comment (mahebbar) [inline cmd/mayactl/app/command/snapshot/revert.go]: and only if there is an issue....current state of data.

File: cmd/mayactl/app/command/snapshot/snapshot.go [hunk 7]
```diff
@@ -20,11 +20,28 @@ import (
...
+var (
+	snapshotCommandHelpText = `
+Usage: mayactl  snapshot <subcommand> [options] [args]
+
+This command provides operations related to a snapshot.
+
+Create a snapshot:
+$ mayactl snapshot create --volname <vol> --snapname <snap>
+
+List snapshot:
```
Comment (mahebbar) [inline cmd/mayactl/app/command/snapshot/snapshot.go]: List snapshots:

**Comment by kmova** [comment]
Fixed by https://github.com/openebs/maya/pull/366

**Pull request closed without merging**